src/controller/CHIPDeviceController.h

```diff
@@ -654,6 +663,14 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
 
     static void OnSessionEstablishmentTimeoutCallback(System::Layer * aLayer, void * aAppState);
 
+    /* This function sends a Device Attestation Certificate chain request to the device.
+       The function does not hold a reference to the device object.
+     */
+    CHIP_ERROR SendCertificateChainRequestCommand(Device * device, CertificateChainType certificateChainType);
+    /* This function sends a Device Attestation Certificate chain request to the device.
```

Comment:
Comment is cut'n'paste from above. Need to update.

Also, argument list should include the nonce argument. It should not be assumed that Device holds it. That way, some external entity could chose to call this method rather than using the built-in flows (when this is more easily supported)

Reply:
will update